docs/getting-started/first-project.md

```diff
@@ -0,0 +1,327 @@
+---
+id: first-project
+title: Your First Project
+---
+
+Welcome to SCSS Starter.
+
+This tutorial introduces you to essentials of SCSS Starter by walking you through building a simple card.
+
+> New To Web Development?<br><br>
+> There are many resources to complement the SCSS Starter docs. Mozilla's MDN docs includes [css](https://developer.mozilla.org/en-US/docs/Web/CSS) introductions. Get Bem docs includes [bem](http://getbem.com/introduction/) introductions.
```

Comment:
Burada bem yerine BEMIT ye yönlendirmeliyiz. 
https://csswizardry.com/2015/08/bemit-taking-the-bem-naming-convention-a-step-further/